modules/index/src/integrationTest/java/org/apache/ignite/internal/index/ItBuildIndexTest.java

```diff
@@ -339,10 +339,10 @@ private static Map<Integer, List<Ignite>> waitForIndexBuild(String tableName, St
                 for (int partitionId = 0; partitionId < internalTable.partitions(); partitionId++) {
                     RaftGroupService raftGroupService = internalTable.partitionRaftGroupService(partitionId);
 
-                    Stream<Peer> allPeers = Stream.concat(Stream.of(raftGroupService.leader()), raftGroupService.peers().stream());
```

Comment:
Do you know why originally there was a leader in concatenation?

Reply:
No, but leader is also presented in the peers' list, so seems like this code were inefficient.